service/app/queries/receive_log_test.go

```diff
@@ -1,65 +0,0 @@
-package queries_test
-
-import (
-	"testing"
-
-	"github.com/boreq/errors"
-	"github.com/planetary-social/scuttlego/di"
-	"github.com/planetary-social/scuttlego/service/app/queries"
-	"github.com/stretchr/testify/require"
-)
-
-func TestReceiveLog(t *testing.T) {
```

Comment:
I think that we could leave a test for the `queries.ReceiveLog` constructor here.

Reply:
[5695852](https://github.com/planetary-social/scuttlego/pull/28/commits/56958528455b55449495dcdd27c9e04a9f004329)